ajax/libs/Snowstorm/package.json

```diff
@@ -8,10 +8,7 @@
     "xmas"
   ],
   "homepage": "http://www.schillmania.com/projects/snowstorm/",
-  "license": {
-    "name": "BSD License",
-    "url": "https://github.com/scottschiller/Snowstorm/blob/master/license.txt"
-  },
+  "license": "BSD License",
```

Comment:
Please update this to SPDX format and give the reference (https://github.com/scottschiller/Snowstorm/blob/master/license.txt in this case) in commit message.
It would be like
```js
"license": "BSD-3-Clause"
```

Reference lists:
- [spdx license list](https://spdx.org/licenses/)
- [cdnjs license list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json)